index.html

```diff
@@ -585,20 +585,20 @@ <h1 class="text-xs-center">Bio</h1>
                     <div class="col-lg-6">
                         <p class="about-text"><strong>Sarah Grant is a journalist with 10+ years of experience.</strong>
                             A recent graduate of Columbia University's data science program, she contributes data
-                            analysis and interactive visualizations for media companies.</p>
+                            analysis and consulting for music companies. She is also the founder of <a href="https://www.sarahgrantedits.com/">Sarah Grant Edits</a>, a digital editing service that helps non-professional writers break into journalism.</p>
```

Comment:
Add target blank so that this also jumps to a new tab?

Better to use `http` not `https` in `<a>` links.

```suggestion
                            analysis and consulting for music companies. She is also the founder of <a target="_blank" href="http://www.sarahgrantedits.com/">Sarah Grant Edits</a>, a digital editing service that helps non-professional writers break into journalism.</p>
```